data/simple/ballots/submitted_ballot_03a29d15-667c-4ac8-afd7-549f19b8e4eb.json

```diff
@@ -0,0 +1,185 @@
+{
```

Comment:
These files are auto generated from the `electionguard` repository. These files should not be placed here.